vms/avm/txs/mempool/mempool.go

```diff
@@ -147,7 +149,7 @@ func (m *mempool) Add(tx *txs.Tx) error {
 	m.numTxs.Inc()
 
 	// Mark these UTXOs as consumed in the mempool
-	m.consumedUTXOs.Union(inputs)
+	m.consumedUTXOs.Put(txID, inputs)
```

Comment:
nit I like to show the var is purposely dropped
```suggestion
	_ = m.consumedUTXOs.Put(txID, inputs)
```

Reply:
I know we enforce this for returned errors... But I feel like it really doesn't add much in cases like this 🤷